test/unit/errors/error-collector.test.js

```diff
@@ -1433,191 +1558,209 @@ describe('Errors', function () {
         let api
-
-        beforeEach(function () {
+        t.beforeEach(() => {
           api = new API(agent)
         })
 
-        it('should contain intrinsic parameters', function () {
+        t.test('should contain intrinsic parameters', (t) => {
           const error = new Error('some error')
           const nowSeconds = Date.now() / 1000
           api.noticeError(error)
 
-          const attributes = getFirstEventIntrinsicAttributes(aggregator)
-          expect(attributes).to.be.a('Object')
-          expect(attributes.type).equal('TransactionError')
-          expect(attributes['error.class']).to.be.a('string')
-          expect(attributes['error.message']).to.be.a('string')
-          expect(attributes.timestamp).closeTo(nowSeconds, 1)
-          expect(attributes.transactionName).equal('Unknown')
+          const attributes = getFirstEventIntrinsicAttributes(aggregator, t)
+          t.ok(typeof attributes === 'object')
+          t.equal(attributes.type, 'TransactionError')
+          t.ok(typeof attributes['error.class'] === 'string')
+          t.ok(typeof attributes['error.message'] === 'string')
+          t.ok(Math.abs(attributes.timestamp - nowSeconds) <= 1)
+          t.equal(attributes.transactionName, 'Unknown')
+          t.end()
         })
 
-        it('should set transactionName to Unknown', function () {
+        t.test('should set transactionName to Unknown', (t) => {
           const error = new Error('some error')
           api.noticeError(error)
 
-          const attributes = getFirstEventIntrinsicAttributes(aggregator)
-          expect(attributes.transactionName).equal('Unknown')
+          const attributes = getFirstEventIntrinsicAttributes(aggregator, t)
+          t.equal(attributes.transactionName, 'Unknown')
+          t.end()
         })
 
-        it('should contain expected attributes, with filter rules', function () {
+        t.test('should contain expected attributes, with filter rules', (t) => {
           agent.config.attributes.enabled = true
           agent.config.attributes.exclude = ['c']
           agent.config.emit('attributes.exclude')
           const error = new Error('some error')
           let customAttributes = { a: 'b', c: 'ignored' }
           api.noticeError(error, customAttributes)
 
-          const agentAttributes = getFirstEventAgentAttributes(aggregator)
-          customAttributes = getFirstEventCustomAttributes(aggregator)
+          const agentAttributes = getFirstEventAgentAttributes(aggregator, t)
+          customAttributes = getFirstEventCustomAttributes(aggregator, t)
 
-          expect(Object.keys(customAttributes)).length(1)
-          expect(customAttributes.c).to.be.undefined
-          expect(Object.keys(agentAttributes)).length(0)
+          t.equal(Object.keys(customAttributes).length, 1)
+          t.notOk(customAttributes.c)
+          t.equal(Object.keys(agentAttributes).length, 0)
+          t.end()
         })
+        t.end()
```

Comment:
I'm sure this is ok but we typically just do `t.autoend()` at the top of the nested test suite

Reply:
Done!